src/components/PostVolunteer/index.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Col } from "react-bootstrap";
+import "./index.css";
+
+export default function PostVolunteer(postInfo) {
+  const { t } = useTranslation();
+
+  if (
+    postInfo.postInfo &&
+    postInfo.postInfo.date &&
+    postInfo.postInfo.title &&
+    postInfo.postInfo.image
+  ) {
+    return (
+      <Col className="singlePost" lg={3} sm={12} md={6}>
+        <p className="cardTitle text-center">{postInfo.postInfo.title}</p>
+        <p>Date: {postInfo.postInfo.date}</p>
+        <p>By : {postInfo.postInfo.profileName}</p>
+        <p className="cardBody text-center">{postInfo.postInfo.description}</p>
+        <img
+          className="cardImg"
+          src={postInfo.postInfo.image}
+          alt={t("postPage.altVolunteerHeader")}
+        />
+      </Col>
+    );
+  }
+  return <> </>;
```

Comment:
What does this return, does it make sense to return loading, until the page loads?

Reply:
I added it because when there was only if and no else it gave an error that the function does not return anything. I changing it to loading gif makes more sense